### PR TITLE
Make stripHandlerT and subHelper available for public use

### PR DESCRIPTION
**Pull request by andrewthad**
I haven't compiled this to check it, but I'm pretty sure it works. I assume that Travis should catch any compile errors if it's broken.

### CODE REVIEW

**Comment by andrewthad** [comment]
https://github.com/yesodweb/yesod/issues/1002

**Comment by MaxGabriel** [comment]
Since these functions are being made public, it'd be nice if `stripHandlerT` was haddock-documented, and potentially the `subHelper` function documentation improved (the current documentation kind of indicates its just for internal use).

**Comment by andrewthad** [comment]
@MaxGabriel That's a very good point. I will add some documentation to them.

**Comment by andrewthad** [comment]
@MaxGabriel I've improved the documentation (although it's still pretty lackluster). Let me know if you have any suggestions.

**Comment by andrewthad** [comment]
This should be ready to go now

File: yesod-core/Yesod/Core/Dispatch.hs [hunk 1]
```diff
@@ -30,6 +30,7 @@ module Yesod.Core.Dispatch
     , defaultMiddlewaresNoLogging
       -- * WAI subsites
     , WaiSubsite (..)
+    , module YCCD
```
Comment (snoyberg) [inline yesod-core/Yesod/Core/Dispatch.hs]: Why not just export `subHelper` directly?

Comment (andrewthad) [inline yesod-core/Yesod/Core/Dispatch.hs]: Good point. I've now changed it to do that instead.

**Comment by snoyberg** [comment]
Thanks for fixing that, I'll just wait for Travis to confirm and then this is good to go.

**Pull request merged**